### PR TITLE
Massive s/editor/inspector

### PR DESCRIPTION
**Pull request by fernandojsg**
Rename all the variables, events, attributes and so on.

### CODE REVIEW

**Comment by dmarcos** [comment]
there are conflicts

**Comment by fernandojsg** [comment]
@dmarcos sorry, fixed!

File: src/actions/entity.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var editor = require('../lib/editor.js');
+var inspector = require('../lib/inspector.js');
```
Comment (ngokevin) [inline src/actions/entity.js]: If inspector is a singleton, might want to cap it.

`INSPECTOR.selectEntity`

Comment (fernandojsg) [inline src/actions/entity.js]: @ngokevin good point, changed!

**Comment by ngokevin** [comment]
r+ from me

**Comment by fernandojsg** [comment]
Thanks!

**Pull request merged**